### DISCUSSION TITLE
Plugin/Scripts Directory

### DISCUSSION THREAD

**Discussion by miranyx** [davatorium/rofi#1823] [Q&A] [answered]
Greetings,

May I inquire as to the appropriate location for installing modes plugins and user scripts?

**Comment by DaveDavenport** [comment, marked as answer]
Yes you may, but I assume that is not the actual question you want to ask.

* C plugins need to be installed into the same prefix as rofi.  Or in a path set by ROFI_PLUGIN_PATH
* user scripts needs to be in your $PATH.

If you are on recent rofi (after june 22) all user scripts in `~/.config/rofi/scripts/` will be auto-loaded.
